### PR TITLE
CONTRIBUTING: allow external pull-requests

### PR DESCRIPTION
**Pull request by kusma**
We now have a CLA in order, which should be the last hurdle against
allowing external pull-requests. So let's remove this section, so
people won't be scared away.

### CODE REVIEW

**Comment by eeue56** [comment]
Maybe worth adding a link to the CLA in this document?

**Comment by kusma** [comment]
@eeue56: good idea!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -112,3 +111,5 @@ members of the project's leadership.
 This Code of Conduct is adapted from the
 [Contributor Covenant](http://contributor-covenant.org), version 1.4,
 available at <http://contributor-covenant.org/version/1/4>
+
+[1]: https://gist.github.com/kusma/2348c1d8e1ab31bcd2d657a5bdb8a1f6#file-fusetools-individual-cla-md
```
Comment (eeue56) [inline CONTRIBUTING.md]: shouldn't this link be https://cla-assistant.io/fusetools/fuselibs-public?

Comment (kusma) [inline CONTRIBUTING.md]: @eeue56: Good point, fixed!

**Pull request merged**